### ISSUE TITLE
writing a tutorial paper on intro to R^2 measures and their computation with `performance` package

### ISSUE THREAD

**Issue by IndrajeetPatil** [easystats/performance#343]

**Comment by bwiernik** [comment]
For recent discussion #332 #334

**Comment by IndrajeetPatil** [comment]
Hi @mkshaw! 👋 

Do you think you would be interested in helping with this? 

Should be right down your alley 🙃

**Comment by mkshaw** [comment]
@IndrajeetPatil it does look like it's down my alley! I recently published [this](https://link.springer.com/article/10.3758/s13428-022-01841-4) tutorial paper on R2s in MLMs using examples to show how to calculate and interpret output. It could be useful to have something similar for the various R2s available in `performance`: a top-level summary of all the measures available and generally how they differ/why we offer them, and then examples one-by-one showing their computation and interpretation.

What are your thoughts on this structure, and @bwiernik based on your discussions in #332 and #334 it seems like you're quite familiar with the measures already, would you be interested in collaborating on this?

**Comment by IndrajeetPatil** [comment]
@mkshaw Are you still interested in working on this? Thanks.

**Comment by mkshaw** [comment]
@IndrajeetPatil I am still interested. I have my dissertation proposal next week, then can flesh out the outline. What software do you recommend for collaborating on this paper? I usually just write in Word, but if I'm starting a pull request and it's larger-scale collaboration, I imagine something else might be prefereable.

**Comment by IndrajeetPatil** [comment]
@mkshaw Would you be comfortable collaborating on GitHub itself? You can have a look at [this PR](https://github.com/easystats/report/pull/342) where we are collaborating to write a new paper.

As for software, we can use RMarkdown to generate a PDF (probably via [rticles](https://pkgs.rstudio.com/rticles/)) we can use for journal submission.

**Comment by mkshaw** [comment]
@IndrajeetPatil Happy to do that, I just need some help getting started, haven't done that before. Looking at the first commit from that PR, did you write the outline in RMarkdown locally, export to pdf, then upload the Rmd, md, and pdf files manually? And then people edit the md from there?

**Comment by strengejacke** [comment]
I'd suggest using Word, since any revisions usually need to be submitted as "tracked changed" anyway, and "tracked changes" feature isn't really comfortable with markdown, even not for collaboration.

**Comment by mkshaw** [comment]
@strengejacke okay so you recommend pull the Word doc from GitHub, edit locally, push?

**Comment by strengejacke** [comment]
Yes, or even a Google docs file (or Word online). It depends on your workflow, pushing/pulling from GitHub is ok, too, I think.

**Comment by mkshaw** [comment]
Does anyone know how to share a google doc file that has people request access to view? I don't want to create an anyone-can-edit link, but I don't have your email addresses to send specific access. It would be best to create a link you can click to request access, but that doesn't seem possible.

**Comment by mattansb** [comment]
If you share the url as-is (say, copied from the browser) it should have that effect.

**Comment by mkshaw** [comment]
Let me know if this works, barebones outline: https://docs.google.com/document/d/1AUWxoTyvRdDA-gjvnWNRlbh5DDZ6dYug/edit?rtpof=true

**Comment by strengejacke** [comment]
Closing some old issues for inactivity. Re-open if this should still be addressed.
